src/Dazinator.Extensions.Options.Updatable/JsonUpdatableOptions.cs

```diff
@@ -8,68 +8,83 @@ namespace Dazinator.Extensions.Options.Updatable
     public class JsonUpdatableOptions<TOptions> : IUpdatableOptions<TOptions>
         where TOptions : class, new()
     {
-        private readonly IOptionsMonitor<TOptions> _monitor;
+        // ##############################################################################################################################
+        // Properties
+        // ##############################################################################################################################
 
-        // private IOptionsSnapshot<TOptions> _options;
-        private readonly IJsonStreamProvider<TOptions> _jsonFileStreamProvider;
-       // private readonly IOptionsMonitorCache<TOptions> _cache;
-        private readonly string _sectionName;
-        private readonly bool _leaveOpen;
-        private readonly static JsonSerializerOptions _defaultSerializerOptions = new JsonSerializerOptions() { IgnoreNullValues = true, WriteIndented = true };
+        #region Properties
 
-        public TOptions Value => _monitor.CurrentValue;
+        // ##########################################################################################
+        // Public Properties
+        // ##########################################################################################
 
-        public TOptions Get(string name) => _monitor.Get(name);
+        public TOptions Value => _Monitor.CurrentValue;
+
+        public TOptions Get(string name) => _Monitor.Get(name);
+
+        // ##########################################################################################
+        // Private Properties
+        // ##########################################################################################
+
+        private readonly IOptionsMonitor<TOptions> _Monitor;
+
+        private readonly IJsonStreamProvider<TOptions> _JsonFileStreamProvider;
+        private readonly string _SectionName;
+        private readonly JsonSerializerOptions _JsonSerializerOptions;
+
+        #endregion
+
+        // ##############################################################################################################################
+        // Constructor
+        // ##############################################################################################################################
+
+        #region Constructor

```

Comment:
Please remove the regions - I prefer the structure of classes to be implicit following conventions, rather than explicit using regions and comment blocks to denote structure..  I usually find regions can be helpful in very large classes with lots of groupings of methods, but if they are that large to warrant regions then that can be a sign of a code smell in that the class might be able to be refactored out as it has too many responsibilities (not always). In this case though most of these regions have one or two methods and just denote structure.